### PR TITLE
Update docs for multi-instance setup

### PR DESCRIPTION
**Pull request by sbrunner**
Related to: https://github.com/camptocamp/rudaz_geomapfish/pull/1
Replace: https://github.com/camptocamp/c2cgeoportal/pull/1372

### CODE REVIEW

File: doc/integrator/create_application.rst [hunk 1]
```diff
@@ -490,15 +507,16 @@ Result
...
+* ``mandant/static/css/proj-<instance>.css``
 * ``vars_<instance>.yaml``
 
-Then run the make command with the ``.mk`` file for the instance you want to setup:
+Then run the make command for the user/instance you want to setup:
 
 .. prompt:: bash
 
-    make <user>_<instance>.mk build
+    ENV=<user> make -f rudaz.mk user_mk
```
Comment (asaunier) [inline doc/integrator/create_application.rst]: Shouldn't you use a more neutral filename than rudaz.mk?

Comment (sbrunner) [inline doc/integrator/create_application.rst]: :-)

**Comment by asaunier** [comment]
LGTM.
@tsauerwein OK for you since you are the author of the original PR?

**Comment by tsauerwein** [comment]
For "2. Create the `<instance>.mk` files": `INSTANCE_ID` should also be set, otherwise you can not run `make -f <instance>.mk build`.

         INSTANCE_ID = <instance>
         INSTANCE = <instance>
         VARS_FILE = vars_$(INSTANCE).yaml
         include <package>.mk

**Comment by sbrunner** [comment]
@tsauerwein you shouldn't run `make -f <instance>.mk` you should run  `make -f <user>_<instance>.mk`, where user is the username or demo, prod, ...

**Comment by sbrunner** [comment]
For the `viewer` it's already done in the point 7.

**Comment by tsauerwein** [comment]
See the comments about `${request.registry.settings['viewer'][*]`, besides it seems to work like that.

**Comment by sbrunner** [comment]
Effectively, thanks @tsauerwein

**Pull request merged**